src/repo.rs

```diff
@@ -1778,6 +1779,18 @@ impl Repository {
             Ok(())
         }
     }
+
+    /// Perform a cherrypick
+    pub fn cherrypick(&self, commit: &Commit, options: Option<CherrypickOptions>) -> Result<(), Error> {
```

Comment:
Typically for these sorts of options the library takes `Option<&mut Options>` to allow re-using them over time.